o11y/metrics/datadog/datadog.go

```diff
@@ -115,55 +120,64 @@ func (d Sink) MetricCleaned(succeed bool, kind string, tags []string) error {
 	return d.metricWithStatus(d.prefix+"cleaned", t)
 }
 
-// MetricReconcile increment reconcile metric
+// MetricReconcile is used to count how many times the controller enters any reconcile loop
 func (d Sink) MetricReconcile() error {
 	return d.metricWithStatus(d.prefix+"reconcile", []string{})
 }
 
-// MetricReconcileDuration send timing metric for reconcile loop
+// MetricReconcileDuration is used at the end of every reconcile loop to indicate the duration that Reconcile() call spent
 func (d Sink) MetricReconcileDuration(duration time.Duration, tags []string) error {
 	return d.timing(d.prefix+"reconcile.duration", duration, tags)
 }
 
-// MetricCleanupDuration send timing metric for cleanup duration
+// MetricCleanupDuration indicates the duration between a Disruption's deletion timestamp, and when the chaos-controller
+// removes its finalizer
 func (d Sink) MetricCleanupDuration(duration time.Duration, tags []string) error {
 	return d.timing(d.prefix+"cleanup.duration", duration, tags)
 }
 
-// MetricInjectDuration send timing metric for inject duration
+// MetricInjectDuration indicates the duration between a Disruption's creation timestamp, and when it reaches a status
+// of Injected, indicating all chaos-injector pods have injected into their targets, and we've reached the expected count
 func (d Sink) MetricInjectDuration(duration time.Duration, tags []string) error {
 	return d.timing(d.prefix+"inject.duration", duration, tags)
 }
 
-// MetricDisruptionCompletedDuration sends timing metric for entire disruption duration
+// MetricDisruptionCompletedDuration indicates the duration between a Disruption's creation timestamp, and when the chaos-controller
+// removes its finalizer
 func (d Sink) MetricDisruptionCompletedDuration(duration time.Duration, tags []string) error {
 	return d.timing(d.prefix+"disruption.completed_duration", duration, tags)
 }
 
-// MetricDisruptionOngoingDuration sends timing metric for disruption duration so far
+// MetricDisruptionOngoingDuration indicates the duration between a Disruption's creation timestamp, and the current time.
+// This is emitted approximately every one minute
 func (d Sink) MetricDisruptionOngoingDuration(duration time.Duration, tags []string) error {
 	return d.timing(d.prefix+"disruption.ongoing_duration", duration, tags)
 }
 
-// MetricPodsCreated increment pods.created metric
+// MetricPodsCreated is used every time the chaos-controller finishes sending a Create request to the k8s api to
+// schedule a new chaos-injector pod. The `succeed` bool argument is false if there was an error returned.
 func (d Sink) MetricPodsCreated(target, instanceName, namespace string, succeed bool) error {
 	status := boolToStatus(succeed)
 	tags := []string{"target:" + target, "disruptionName:" + instanceName, "status:" + status, "namespace:" + namespace}
```

Comment:
namespace here is the chaos namespace or the disruption namespace?

Reply:
great catch, i'll look

Reply:
it's the disruption namespace, which is not the chaos pod's namespace. fixing